src/utils/requests.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from "@angular/core";
+import { Headers, Http } from "@angular/http";
+
+@Injectable()
+export class Requests {
+
+	private headers = new Headers({'Content-Type': 'application/json'});
+	
+	constructor(private http: Http) { }
```

Comment:
Angular2 simply takes over to the constructor. Looks like I have to find a way around.